src/Symfony/Component/Notifier/Message/SentMessage.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Message;
+
+/**
+ * @author Jérémy Romey <jeremy@free-agent.fr>
+ *
+ * @experimental in 5.2
+ */
+class SentMessage
+{
+    private $original;
+    private $transport;
+    private $messageId = null;
```

Comment:
` = null` is the default, so it can be removed

Reply:
Fixed